AFQ/definitions/utils.py

```diff
@@ -72,9 +72,12 @@ def _arglist_to_string(args, get_attr=None):
 def name_from_path(path):
     file_name = op.basename(path)  # get file name
     file_name = drop_extension(file_name)  # remove extension
-    if "-" in file_name:
-        file_name = file_name.split("-")[-1]  # get suffix if exists
-    return file_name
+    if "desc-" in file_name:  # get desc if exists
```

Comment:
I think that it would be helpful to write a docstring for this function (and tests?), possibly even with examples, because I am still not sure if this does what it's supposed to do.

Reply:
It's trying to find the name for the scalar, which will be used in the final CSV after profiling, from the file path

Reply:
And this would have produced strange names before, because when there is a session identifier in the filename everything after the dash gets concatenated in? Maybe I can propose a test that fails before the change.